src/poetry/console/application.py

```diff
@@ -306,10 +306,13 @@ def _configure_installer(self, command: InstallerCommand, io: IO) -> None:
         installer.use_executor(poetry.config.get("experimental.new-installer", False))
         command.set_installer(installer)
 
-    def _load_plugins(self, io: IO) -> None:
+    def _load_plugins(self, io: IO = None) -> None:
         if self._plugins_loaded:
             return
 
+        if io is None:
+            io = NullIO()
+
```

Comment:
This seems to be only required for the test. Not sure, if it's better to just pass NullIO() in the test and don't introduce a default.

Reply:
Was on the fence on this one, I am okay either way, but figured we will eventually need this anyway and since the method strictly didn't require a non NullIO, figure it reasonable.

But yeah, will cover it when I do the next review sweep for this one.

Reply:
Looked through the other cases where this is done, and since the factory allready has a similar pattern employed, I am inclined to keep it as is for now unless there is a strong objection on this.